ddl/ddl_db_test.go

```diff
@@ -369,6 +369,8 @@ LOOP:
 	// Make sure there is index with name c3_index.
 	c.Assert(nidx, NotNil)
 	c.Assert(nidx.Meta().ID, Greater, int64(0))
+	ctx.Txn().Rollback()
```

Comment:
Add it to `testDropIndex` is ok, but I find `ctx.NewTxn()` don't return error, so it can work without it.
I don't know what cause the difference.